portfolio-project-matching-app/pages/browseProjects.js

```diff
@@ -55,18 +58,18 @@ const browseProjects = () => {
         let newVisible = visibleProjects
 
         // loop through hidden projects and potentially add additional filter
-        for (const project of hiddenProjects) {
-            project.addFilter(choice)
+        for (const hiddenProject of hiddenProjects) {
+            hiddenProject.addFilter(choice)
```

Comment:
Tried to make my variables a little more descriptive in a couple places. I did not like how I had some `project.project.name` showing up in places. Figured `hiddenProject.project.name` looks a tiny bit better.